### PR TITLE
Feature#104 design student upcoming sessions list item view

### PR DESCRIPTION
**Pull request by gregpmillr**
See commits. 
Renamed files as per issue comment request
Showing specific tutoring session from student home view
Cancellation functionality included on specific tutoring session

TODO:
Location button for map

### CODE REVIEW

**Comment by gregpmillr** [comment]
See issue #104

File: app/src/main/res/layout/student_home_upc_session_details.xml [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        android:layout_centerHorizontal="true"
+        android:layout_marginTop="19dp" />
+
+
+    <ImageView
+        android:id="@+id/student_upc_details_tutor_picture"
+        android:layout_width="60dp"
+        android:layout_height="60dp"
+        android:layout_marginRight="6dip"
+        android:src="@drawable/egg1"
```
Comment (Mons1Oerjan) [inline app/src/main/res/layout/student_home_upc_session_details.xml]: Change default image to "default_icon". It got pushed into the master branch today, so if you don't have it yet, you should pull master into this branch

Comment (gregpmillr) [inline app/src/main/res/layout/student_home_upc_session_details.xml]: Added!

**Comment by Mons1Oerjan** [comment]
Approved!

**Pull request merged**